$OpenBSD: patch-config-scripts_cups-pdf_m4,v 1.4 2012/02/17 07:45:10 ajacoutot Exp $

CUPS check for the ghostscript binary; we cannot depend on
print/ghostscript/gnu because of a cyclic dependency but we know our
gs supports the ps2write device.

--- config-scripts/cups-pdf.m4.orig	Fri Feb 17 08:35:39 2012
+++ config-scripts/cups-pdf.m4	Fri Feb 17 08:35:59 2012
@@ -91,12 +91,8 @@ if test "x$CUPS_PDFTOPS" != x; then
 	DEFAULT_PDFTOPS=""
 elif test "x$CUPS_GHOSTSCRIPT" != x; then
 	AC_MSG_CHECKING(whether gs supports the ps2write device)
-	if ($CUPS_GHOSTSCRIPT -h 2>&1 | grep -q ps2write); then
-		AC_MSG_RESULT(yes)
-		AC_DEFINE(HAVE_GHOSTSCRIPT_PS2WRITE)
-	else
-		AC_MSG_RESULT(no)
-	fi
+	AC_MSG_RESULT(yes)
+	AC_DEFINE(HAVE_GHOSTSCRIPT_PS2WRITE)
 
 	DEFAULT_PDFTOPS=""
 else
